### PR TITLE
HRIS-12 [FE] Integrate login functionality

### PR DESCRIPTION
**Pull request by NiloJr-sun**
## Issue Link

https://framgiaph.backlog.com/view/HRIS-12

## Definition of Done

- [x] Users can log in using their Google Company Accounts

## Notes
 - Setup of the authentication of user is temporary, it will be restructured on the next incoming sprints

## Pre-condition
- Go to the root directory and run ``` docker compose up --build ```
- Run ``` cd client ```
- You can run ``` npm i ``` manually just to make sure all dependencies are met
- Pre-existing user in the Users table with a valid Google Company Account

## Expected Output
- Upon log in, it will ask the user to login via their Google Account
- After a series of steps, it will verify if the users email account is verified by the company
- If the email is valid, the user will be redirected to the index page
- If the email is invalid, the user will be redirected to the sign in page
- Session tokens will also be saved on the ``` Personal_Access_Token ``` table

## Screenshots/Recordings

Title
Image/Picture

https://user-images.githubusercontent.com/109492180/212038026-b0921ff0-f811-43fa-be8f-b012de315bab.mp4

Saved token:

image

### CODE REVIEW

File: client/.env [hunk 1]
```diff
@@ -0,0 +1,10 @@
+GOOGLE_CLIENT_ID=754182487413-jgql369a3bppphd03bimol67q2g880l6.apps.googleusercontent.com
+GOOGLE_CLIENT_SECRET=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
+
```
Comment (rogeliojohnoliverio) [inline client/.env:3]: sir please exclude this from your pr

Comment (NiloJr-sun) [reply client/.env:3]: Okay sir

**Review by jsvelte — APPROVED** [review]
# Approved 💯

**Review by rogeliojohnoliverio — APPROVED** [review]
Approved

**Review by arden-dave — APPROVED** [review]

**Comment by arden-dave** [comment]
Approved

**Pull request merged**